adminSiteServer/apiRouter.ts

```diff
@@ -883,11 +951,13 @@ deleteRouteWithRWTransaction(
             [chart.id]
         )
 
-        const row = await db.knexRawFirst<{ configId: number }>(
+        const row = await db.knexRawFirst<Pick<DbPlainChart, "configId">>(
             trx,
             `SELECT configId FROM charts WHERE id = ?`,
             [chart.id]
         )
+        if (!row)
```

Comment:
I think `configId` is nullable in the DB, meaning we should also check `row.configId`

Reply:
I think that used to be the case in an earlier draft of Sophia's but it's now not nullable. But the check doesn't hurt so I'll add it here in case we change our mind again in the future